src/features/showReferences.ts

```diff
@@ -39,6 +40,11 @@ export class ShowReferencesFeature implements StaticFeature {
       return;
     }
 
+    const codeLensReferenceCount = config('terraform').get<boolean>('codelens.referenceCount');
```

Comment:
This one should be opt-in and not enabled by default, right? So we have to use a default value of `false` here or update the `if` next line to a more "falsy" check